docs/api_docs/maintenance/scale/horizontal-scale.md

```diff
@@ -0,0 +1,467 @@
+---
+title: Horizontal Scale
+description: How to scale a cluster, scale replicas, scale instances
+keywords: [horizontal scaling, Horizontal Scale]
+sidebar_position: 2
+sidebar_label: Horizontal Scale
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Horizontal Scale
+
+From v0.9.0, the horizontal scale can be divided into two types to better support various scenarios. Both final-state-oriented and procedure-oriented scale support scale in and out, but what distinguishes them is that the final-state-oriented scale only scales replicas and the procedure-oriented one supports scaling both the replicas and instances.
+
+- Final-state-oriented
```

Comment:
"Final-state-oriented" and "Procedure-oriented" are hard to understand; it would be better to replace them with more widely accepted expressions.

"Final-state-oriented" can be expressed as "Modifying the Cluster in a declarative API style".
"Procedure-oriented" can be expressed as "Creating an OpsRequest".